metals/src/main/scala/scala/meta/internal/metals/callHierarchy/CallHierarchyItemInfo.scala

```diff
@@ -0,0 +1,16 @@
+package scala.meta.internal.metals.callHierarchy
+
+/**
+ * Data that is preserved between a call hierarchy prepare and incoming calls or outgoing calls request.
+ *
+ * @param symbols The set of symbols concerned by the request.
+ * @param visited Symbols that are already visited to handle recursive cases.
+ * @param isLocal Do we have to search for symbols in the whole workspace.
+ * @param searchLocal Do we have to search for symbols locally.
```

Comment:
`isLocal` and `searchLocal`, if I understand correctly models 3 cases? 
- search only local scope
- search the whole workspace
- search the whole workspace except local scope

Am I correct?

Reply:
Yes, this is inspired from code in ReferenceProvider, so the associated comment is : 

```scala
/* search local in the following cases:
     * - it's local symbol
     * - it's a dependency source.
     *   We can't search references inside dependencies so at least show them in a source file.
     * - it's a standalone file that doesn't belong to any build target
     */
```